### PR TITLE
Create GKE clusters in separate VPC networks

### PR DESCRIPTION
**Pull request by crabtree**
**Description**

Changes proposed in this pull request:

- Create GKE clusters in separate VPC networks

**Related issue(s)**

### CODE REVIEW

File: prow/scripts/cluster-integration/helpers/create-network-with-subnet.sh [hunk 1]
```diff
@@ -22,4 +23,8 @@ fi
 
 gcloud compute networks create "${GCLOUD_NETWORK_NAME}" \
  --project="${GCLOUD_PROJECT_NAME}" \
- --subnet-mode=custom
\ No newline at end of file
+ --subnet-mode=custom
+
+gcloud compute networks subnets create "${GCLOUD_SUBNET_NAME}" \
+ --network="${GCLOUD_NETWORK_NAME}" \
+ --range=10.0.0.0/24
```
Comment (Demonsthere) [inline prow/scripts/cluster-integration/helpers/create-network-with-subnet.sh]: Please add `--region` for consistency and safety

File: prow/scripts/cluster-integration/helpers/delete-network-with-subnet.sh [hunk 2]
```diff
@@ -20,5 +21,9 @@ if [ "${discoverUnsetVar}" = true ] ; then
...
 fi
 
+gcloud compute networks subnets delete "${GCLOUD_SUBNET_NAME}" \
+ --quiet
+
 gcloud compute networks delete "${GCLOUD_NETWORK_NAME}" \
- --project="${GCLOUD_PROJECT_NAME}"
\ No newline at end of file
+ --project="${GCLOUD_PROJECT_NAME}" \
+ --quiet
```
Comment (Demonsthere) [inline prow/scripts/cluster-integration/helpers/delete-network-with-subnet.sh]: also please specify `--region`

File: prow/scripts/cluster-integration/helpers/create-network-with-subnet.sh (hunk 1 above)
Comment (crabtree) [reply prow/scripts/cluster-integration/helpers/create-network-with-subnet.sh]: done

File: prow/scripts/cluster-integration/helpers/delete-network-with-subnet.sh (hunk 2 above)
Comment (crabtree) [reply prow/scripts/cluster-integration/helpers/delete-network-with-subnet.sh]: done

**Pull request merged**